theano/tensor/nnet/corr_gemm.c

```diff
@@ -203,22 +206,23 @@ PyArrayObject* corrMM(PyArrayObject* bottom,
     char Trans = 'T';
     PyArrayObject *output;
 
-    %(omp_set_threads)s(max_threads);
+    %(set_omp_threads)s(max_threads);
```

Comment:
I forgot why this line is used if we set that max blas threads. What happen if there is a computer with 20 cores and the batch size of 10 and we don't call this line? My first guess is that the 10 extra cores are just not used.

Line 322: in this commit also use %(omp_flags)s, but do  weight_dim[1] iteration and not batchSize. So if batchsize is 1, this line would like the number if threads for that section.

If it is good to keep this line, we should undo this at the end of this  op to let other node in the graph use there own logic.